spec/models/orchestration_stack/retirement_management_spec.rb

```diff
@@ -129,12 +129,12 @@
   it "#raise_retirement_event" do
     event_name = 'foo'
     event_hash = {
-      :orchestration_stack  => @stack,
-      :type                 => "OrchestrationStack",
-      :retirement_initiator => "system"
+      :userid              => nil,
+      :orchestration_stack => @stack,
+      :type                => "OrchestrationStack",
     }
 
-    expect(MiqEvent).to receive(:raise_evm_event).with(@stack, event_name, event_hash, {})
+    expect(MiqEvent).to receive(:raise_evm_event).with(@stack, event_name, event_hash, nil)
```

Comment:
in a number of places, it looks like the empty options changed from `{}` to `nil` - I don't totally see where this happened in this PR

Reply:
Yeah, I need something testing the original not having a valid zone thing now...